include/benchmark/benchmark.h

```diff
@@ -429,6 +429,46 @@ struct Statistics {
       : name_(name), compute_(compute) {}
 };
 
+// Converts whatever the stored manual time is to
+// seconds, which is used to estimate iterations
+// and similar
+typedef double(BenchmarkTimeCostFunc)(double);
+
+struct BenchmarkTime {
```

Comment:
does this need to be part of the public API?

Reply:
It is to me. I want the public API to be very clear, especially when we have to define structures by necessity in the header.

Reply:
Thanks, but no. Let's keep this focused. I'll take care of Statistics.